lib/protobuf/encoder.ex

```diff
@@ -183,15 +183,15 @@ defmodule Protobuf.Encoder do
   def encode_type(:sint64, n) when n >= -0x8000000000000000 and n <= 0x7FFFFFFFFFFFFFFF,
     do: n |> encode_zigzag |> encode_varint
 
-  def encode_type(:fixed64, n) when n >= 0 and n <= 0xFFFFFFFFFFFFFFFF, do: <<n::64-little>>
+  def encode_type(:fixed64, n) when n >= 0 and n <= 0xFFFFFFFFFFFFFFFF, do: [<<n::64-little>>]
 
   def encode_type(:sfixed64, n) when n >= -0x8000000000000000 and n <= 0x7FFFFFFFFFFFFFFF,
-    do: <<n::64-signed-little>>
+    do: [<<n::64-signed-little>>]
 
-  def encode_type(:fixed32, n) when n >= 0 and n <= 0xFFFFFFFF, do: <<n::32-little>>
+  def encode_type(:fixed32, n) when n >= 0 and n <= 0xFFFFFFFF, do: [<<n::32-little>>]
 
   def encode_type(:sfixed32, n) when n >= -0x80000000 and n <= 0x7FFFFFFF,
-    do: <<n::32-signed-little>>
+    do: [<<n::32-signed-little>>]
```

Comment:
Why wrap these binaries in lists?

Reply:
Good catch. I think it was a leftover from when I was trying to prevent improper lists.

Reply:
Fixed.